.editorconfig

```diff
@@ -0,0 +1,16 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.{js,py,xml,java,json,yml}]
```

Comment:
We commonly use `kt` and `yaml` as extensions too

Reply:
OK, I might as well add them then right? I'd imagine people would customise this file for their specific projects so could add/remove as they see fit.